### ISSUE TITLE
3d Charts not working?

### ISSUE THREAD

**Issue by renewsoqv** [MaazAli/Meteor-HighCharts#18]
I'm trying to render a 3d chart but so far not able to.
I'm using the code on http://www.highcharts.com/demo/3d-pie

Any suggestions?

**Comment by jhuenges** [comment]
I guess [this](http://code.highcharts.com/highcharts-3d.js) is not inclueded in this package. If @MaazAli  is still up for the idea (#12), it could be implemented as a separate package.

**Comment by MaazAli** [comment]
Yeah we should be using separate packages for additional highcharts.

**Comment by jhuenges** [comment]
In my opinion it would be good if all of them are in the same "namespace". So it would be great if you could create the packages and repos. And of course I can help with filling them ;)

**Comment by jhuenges** [comment]
bump

**Comment by MaazAli** [comment]
What package names are we looking at here?

**Comment by jhuenges** [comment]
I guess something like ```highcharts-gauge``` and ```highcharts-3d``` etc. And maybe rename this package to ```highcharts-base```?

**Comment by jhuenges** [comment]
@MaazAli I ll start tomorrow :)

**Comment by MaazAli** [comment]
Sounds good!

**Comment by jhuenges** [comment]
@renewsoqv 
We are working on releasing the [new 3d package](https://github.com/MaazAli/highcharts-3d). Shouldnt take too longe :)

**Comment by jhuenges** [comment]
Sorry this takes so long! The package is basically ready, only the publication to atmosphere is missing. If you are in dire need, simply clone the repo and take the code.
